src/WebJobs.Script/Workers/Rpc/RpcWorkerDescription.cs

```diff
@@ -46,6 +47,12 @@ public class RpcWorkerDescription : WorkerDescription
         [JsonProperty(PropertyName = "supportedRuntimeVersions")]
         public List<string> SupportedRuntimeVersions { get; set; }
 
+        /// <summary>
+        /// Gets or sets the regex used for sanitizing the runtime version string.
+        /// </summary>
+        [JsonProperty(PropertyName = "sanitizeRuntimeVersion")]
+        public string SanitizeRuntimeVersion { get; set; }
```

Comment:
rename to `SanitizeRuntimeVersionRegex`?

Reply:
Thanks, I thought about that, will rename.